angular-viewport-watch.js

```diff
@@ -64,12 +72,32 @@
                 function enableDigest() {
                     toggleWatchers(scope, true);
                 }
+
+                // This function updates the listeners to on or off based on whether
+                // this the currently active base state
+                function updateListeners() {
+                  if (!attr.viewportWatch || $state.includes(attr.viewportWatch)) {
+                    elementWatcher.enterViewport(enableDigest);
+                    elementWatcher.exitViewport(disableDigest);
+                  } else {
+                    elementWatcher.off('enterViewport', enableDigest);
+                    elementWatcher.off('exitViewport', enableDigest);
```

Comment:
Does calling off on the elementWatcher turn off watches associated with it?  Will this code turn watches off that are not in the current state?